ui/explore/src/components/ExploreManager/TimeSeriesExplorer.tsx

```diff
@@ -0,0 +1,70 @@
+// Copyright 2024 The Perses Authors
```

Comment:
Since you named the other TracesExplorer, imho this one should be called, or at least contain "Metrics" in it. If I understood well you're preparing the ground to add a new tab for instant querying of metrics, so in the end I see two ways to go here:

1. 3 tabs: "Traces", "Metrics - range", "Metrics - instant"
2. 2 tabs: "Traces" and "Metrics" + a switch between range & instant for metrics within the tab panel

Personally I like the idea to have a first high-level choice to make between traces, metrics & logs, but it would be adding 1 click in the user's journey.

Reply:
Yes solution 2. There will be also some query params to prevent from having to select and allow to share

Reply:
I will change to metrics ;)